pkgs/development/python-modules/pyu2f/default.nix

```diff
@@ -1,34 +1,35 @@
-{ stdenv, lib, fetchFromGitHub, buildPythonPackage,
-  six, mock, pyfakefs, unittest2, pytest
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, six
+, mock
+, pyfakefs
+, pytest
+, unittest2
 }:
 
 buildPythonPackage rec {
   pname = "pyu2f";
-  version = "0.1.4";
+  version = "0.1.5a";
 
   src = fetchFromGitHub {
     owner = "google";
     repo = pname;
     rev = version;
-    sha256 = "0waxdydvxn05a8ab9j235mz72x7p4pwa59pnxyk1zzbwxnpxb3p9";
+    sha256 = "0mx7bn1p3n0fxyxa82wg3c719hby7vqkxv57fhf7zvhlg2zfnr0v";
   };
 
-  # Platform detection for linux fails
-  postPatch = lib.optionalString stdenv.isLinux ''
-    rm pyu2f/tests/hid/macos_test.py
-  '';
-
   propagatedBuildInputs = [ six ];
 
-  checkInputs = [ pytest mock pyfakefs unittest2 ];
+  checkInputs = [ mock pyfakefs pytest unittest2 ];
```

Comment:
Could this be reduced to ``checkInputs = [ pytestCheckHook pyfakefs mock ];``? Could also remove the custom ``checkPhase``.

Unittest2 is only needed for python < 2.7, and mock is only needed for python < 3.3.

Reply:
Addressed in 16c8941c1325a7d61f0ca2258b0b484f854055bb

Reply:
If you're effectively restricting build to python3 (as you seem to be implicitly doing), then it'd be worth removing ``mock`` as well. Might also be worth an explicit ``disabled = pythonOlder "3.3"``?

Reply:
I tried building without mock and it failed.

Reply:
Ah, it's probably an imports issue. e.g. the difference between (the equivalent) ``import unittest.mock`` & ``import mock``.